LICENSE

```diff
@@ -1,10 +1,10 @@
-Copyright (c) 2019-2020 Textile <contact@textile.io>
+The MIT License (MIT)
 
+Copyright (c) 2020 Protocol Labs
```

Comment:
Is this cool ? Do we need to do extra steps ?

Reply:
Hmmm, since there are multiple contributors involved here, we might need to have this as something more like:

```
Copyright (c) 2020 Project contributors
```

But I'm not all that well-versed in the potential issues here. Maybe @andrewxhill has some thoughts?

Reply:
nothing from me, no pref